.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,54 @@
+# Workflow for building and testing eventuals:grpc on
+# macOS, Ubuntu. In future we gonna run this job for
+# Windows.  
+name: Build and Run all tests
+
+on:
+  push:
+    branches:
+      - "main"
+    paths-ignore:
+      - "**.md"
+  pull_request:
+    paths-ignore:
+      - "**.md"
+  
+jobs:
+  build-and-test:
+    name: Build and Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ["macos-latest", "ubuntu-latest"]
```

Comment:
Can you move the comment that we're not using Windows because of a known build error down here please?